developers/pull_request_checklist.md

```diff
@@ -46,3 +46,14 @@ another developer.
 * Are there any comments that haven't been addressed?
 * Did someone "ACK" the PR?
 * Is the PR still green?
+
+# Tips for reviewers
```

Comment:
The text is very new developer focused so many this belongs more so in the becoming a contributor section? As we chatted in IRC, I don't necessarily agree with some of the limitations of this (PRs can be reviewed before tests pass, sometimes design changes things and would make your tests invalid, and reviewing larger PRs that include design changes is something I would encourage -- they may be daunting but you can learn a lot).